docs/cristin.ttl

```diff
@@ -473,21 +501,21 @@
     :isignored   "true" .
 
 :institusjonsnavn_opprinnelig
-    a            rdf:Property ;
+    a            :cristinfield ;
     rdfs:comment "The original name of the institution at the time the Cristin entry was created. Immutable value" ;
     rdfs:range   xsd:string ;
     rdfs:domain  :VARBEID_PERSON_STED_Class ;
     :isignored   "true" .
```

Comment:
I see many of these as "ignored", and can agree wrp/ dereferencing and such, but doesn't NVA in its archive role have use for knowing that an article was published crediting Rikshospitalet, even though Rikshospitalet later became a part of Oslo University Hospital?

Not complaining, just asking. :-)

Reply:
:isignored shows the sitatuation as it is now. My goal is to document what is the meaning of the each field and what we do with this at present. When the product owener, project manager, read it, they will decide whether we need this information or not.